### ISSUE TITLE
Disallow execution on Server

### ISSUE THREAD

**Issue by mploquin** [VeliovGroup/Meteor-logger#3]
Hi,

It is no longer possible to disallow execution on the server, even with the option set to false.

**Comment by dr-dimitru** [comment]
Hello @mploquin ,

Could you give a code sample please. And with what adapter?

**Comment by mploquin** [comment]
Code sample :
```js
import { Logger } from 'meteor/ostrio:logger';
import { LoggerConsole } from 'meteor/ostrio:loggerconsole';

export const log = new Logger();

const loggerConsole = new LoggerConsole(log, {
  format(opts) {
    return `[${opts.level}] - ${opts.message}`;
  },
});
loggerConsole.enable({
  enable: true,
  server: false,
}); 
```

I think this is with all adapters. In the Logger class, the `rule` method seems force the `options.server` to true even if it's defined to false.

**Comment by dr-dimitru** [comment]
As mentioned in this example:
```js
log.rule('AdapterName', {
  enable: true,
  filter: ['ERROR', 'FATAL', 'WARN'],
  client: false, /* Allow to call, but not execute on Client */
  server: true   /* Calls from client will be executed on Server */
});
```

Setting `{server: false}` will disallow to execute calls from Client on Server. As Server is trusted side we have no intent to control or limit calls. All restrictions is made for Client as for not trusted side.

Anyways, what are you trying to accomplish? Calls from Client to Server or direct calls on Server? What should be prohibited?

**Comment by mploquin** [comment]
I do not have the same behavior compared to the previous version (1.1.2). Setting {server: false} or {enable: false} do not seem to have any effect.

**Comment by dr-dimitru** [comment]
> I do not have the same behavior compared to the previous version (1.1.2). Setting {server: false} or {enable: false} do not seem to have any effect.

This is a reason why I bumped up the major version.

 1. What are you trying to accomplish? 
 2. Could you share a code you're using (*and where - Client or Server*), its result, and expected behavior?

**Comment by mploquin** [comment]
For example, when I try to disable an adapter with `{enable: false}`, it is always on. Take a look at the `rule` method of the Logger class (line 130), if` enable` is set to false, it is forced to true.

**Comment by dr-dimitru** [comment]
@mploquin sorry for delay on this one.
I remember about it, but I'm very busy now, sorry, I hope to have a chance to solve it on next week. Stay tuned!

**Comment by dr-dimitru** [comment]
@mploquin I'm sorry for delay. Thank you for reporting this, fixed in [`v2.0.3`](https://github.com/VeliovGroup/Meteor-logger/releases/tag/2.0.3)

Let me know how it works for you
